lib/kvdb/kvdb_rparams.c

```diff
@@ -689,7 +689,7 @@ static const struct param_spec pspecs[] = {
         .ps_bounds = {
             .as_uscalar = {
                 .ps_min = csched_rp_kvset_iter_async,
-                .ps_max = csched_rp_kvset_iter_mcache,
+                .ps_max = csched_rp_kvset_iter_mmap,
```

Comment:
Seems strange to have "rp" in the rparam name...

Reply:
I don't disagree.  This param should not be called `csched_policy` either.  In fact, we should probably just remove the parameter.   Who knows if it even works anymore.

Reply:
But not in this PR.